FreeCAD fails to compile when HAVE_TBB (seems to be an OpenCV flag) is set, which occurs when CMAKE_BUILD_TYPE is set to none.
--- ./src/Mod/Import/App/ImportOCAF.cpp
+++ ./src/Mod/Import/App/ImportOCAF.cpp
@@ -100,12 +100,6 @@
 #include <App/DocumentObject.h>
 #include <App/DocumentObjectGroup.h>
 
-#ifdef HAVE_TBB
-#include <tbb/parallel_for.h>
-#include <tbb/blocked_range.h>
-#include <tbb/task_group.h>
-#endif
-
 using namespace Import;
 
 #define OCAF_KEEP_PLACEMENT
@@ -139,10 +133,6 @@
                             std::vector<App::DocumentObject*>& lValue)
 {
     int hash = 0;
-#ifdef HAVE_TBB
-    using namespace tbb;
-    task_group g;
-#endif
     TopoDS_Shape aShape;
 
     std::vector<App::DocumentObject *> localValue;
@@ -284,10 +274,6 @@
                              std::vector<App::DocumentObject*>& lValue, bool mergeShape)
 {
     const TopoDS_Shape& aShape = aShapeTool->GetShape(label);
-#ifdef HAVE_TBB
-    using namespace tbb;
-    task_group g;
-#endif
 
     if (!aShape.IsNull() && aShape.ShapeType() == TopAbs_COMPOUND) {
         TopExp_Explorer xp;
